DllImportGenerator/DllImportGenerator/StubCodeGenerator.cs

```diff
@@ -174,7 +177,7 @@ public override (string managed, string native) GetIdentifiers(TypePositionInfo
                 AppendVariableDeclations(setupStatements, retMarshaller.TypeInfo, retMarshaller.Generator);
             }
 
-            if (this.dllImportData.SetLastError)
+            if (this.dllImportData.SetLastError && !options.GenerateForwarders())
```

Comment:
Comment on this logic. Why does this matter? Also, I like to see `this.` so it is clean this isn't an argument.

Reply:
Same below.

Reply:
We don't want to generate our own SetLastError logic if we're having the runtime handle the SetLastError logic.

I'll add a comment.